dev-client/src/screens/HomeScreen/components/TemporaryLocationCallout.tsx

```diff
@@ -80,32 +86,30 @@ export const TemporaryLocationCallout = ({
       buttons={<CloseButton onPress={closeCallout} />}
       isPopover={true}>
       <Column mt="12px" space="12px">
-        {!isSoilIdReady && <ActivityIndicator size="small" />}
-        {topSoilMatch && (
-          <>
-            <CalloutDetail
-              label={t('site.soil_id_prediction')}
-              value={topSoilMatch.soilInfo.soilSeries.name}
-            />
-            <Divider />
-            <CalloutDetail
-              label={t('site.ecological_site_prediction')}
-              value={
-                topSoilMatch.soilInfo.ecologicalSite?.name ??
-                t('site.soil_id.soil_info.no_matches')
-              }
-            />
-            <Divider />
-          </>
-        )}
-        {elevation ? (
+        <>
+          <CalloutDetail
+            label={t('site.soil_id_prediction')}
+            value={soilMatchDisplay}
+          />
+          <Divider />
           <CalloutDetail
-            label={t('site.elevation_label')}
-            value={renderElevation(t, elevation)}
+            label={t('site.ecological_site_prediction')}
+            value={ecologicalSiteDisplay}
           />
-        ) : (
-          <ActivityIndicator size="small" />
-        )}
+          <Divider />
+        </>
+        <CalloutDetail
+          label={t('site.elevation_label')}
+          value={
+            elevation ? (
+              <Text bold textTransform="uppercase">
```

Comment:
```suggestion
              <Text bold>
```

Reply:
The [figma](https://www.figma.com/design/KMJbhjVyrrLFHj6jGcX9lN/LandPKS-Design-Prototype-working-file?node-id=5800-14576&t=mHCgGwk3tEiBTkny-0) says "2800 FEET" -- are you sure this is supposed to be lowercase? (Would say "3.03 m" instead of "3.03 M")

Reply:
SI units are generally written in lowercase.